docs/content/docs/engines/hive.md

```diff
@@ -32,20 +32,51 @@ Table Store currently supports the following features related with Hive:
 
 ## Version
 
-Table Store currently supports Hive 2.3.
+Table Store currently supports Hive 2.1, 2.2 and 2.3.
 
 ## Execution Engine
 
 Table Store currently supports MR and Tez execution engine for Hive.
 
 ## Install
 
+### Table Store Hive Catalog (For Flink)
+
+{{< stable >}}
+Download the jar file with corresponding version.
```

Comment:
Add some information to spark3 page too?
Spark3 need this to create table and etc...